gender_novels/corpora/sample_novels/sample_novels.csv

```diff
@@ -87,3 +87,14 @@ author,date,title,country_publication,author_gender,filename,notes
 "Godwin, Francis",1768,"The Strange Voyage and Adventures of Domingo Gonsales, to the World in the Moon",England,male,godwin_voyage.txt,
 "Goldsmith, Oliver",1770,The Deserted Village,England,male,goldsmith_desert.txt,
 "Goulart, Ron",1960,Subject to Change,United States,male,goulart_subject.txt,
+"Eastman, Seth and Mary",1852,Aunt Phillis's Cabin,United States,both,eastman_phillis.txt,
+"Eastwood, Frances",1870,Geoffrey the Lollard,United States,male,eastwood_lollard.txt,
+"Eaton, Walter Prichard",1922,Boy Scouts at Crater Lake,United States,male,eaton_craterlake.txt,
+"Edginton, May",1917,Married Life,,female,edginton_marriedlife.txt,
+"Edholm, Lizetter",1932,The Merriweather Girls and the Mystery of the Queen's Fan,,female,edholm_merriweathergirls.txt,

```

Comment:
Do we not know where Edholm's novel was published? (In that case, you can probably put "Unknown"